mutagen/wave.py

```diff
@@ -0,0 +1,369 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2017  Borewit
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+
+"""Microsoft WAVE/RIFF audio file/stream information and tags."""
+
+import sys
+import struct
+from struct import pack
+
+from ._compat import endswith, reraise
+
+from mutagen import StreamInfo, FileType
+
+from mutagen.id3 import ID3
+from mutagen._riff import RiffFile, RiffChunkHeader
+from mutagen.id3._util import ID3NoHeaderError, error as ID3Error
+from mutagen._util import resize_bytes, delete_bytes, MutagenError, loadfile, \
+    convert_error
+
+__all__ = ["WAVE", "Open", "delete"]
+
+
+class error(MutagenError):
+    pass
+
+
+class InvalidChunk(error):
+    pass
+
+
+def is_valid_chunk_id(id):
+    # looks like this is failing if python is not started with -bb in TravisCI:
+    # assert isinstance(id, text_type)
+
+    return ((len(id) <= 4) and (min(id) >= u' ') and
```

Comment:
That has been derived from  [aiff.py](https://github.com/quodlibet/mutagen/blob/master/mutagen/aiff.py)

If the chunks are indexed including the space padding, the key should be 4 indeed